### ISSUE TITLE
Updating the models?

### ISSUE THREAD

**Issue by FieryVector** [Kxnrl/NeptuniaCSGO#3]
Hi.
Just a small question. Its been a while so, will you update the models?
It looks like in one CSGO Update the changed something and now I get the error missing bones.

Reaper951

**Comment by Kxnrl** [comment]
can u show me what`s wrong?
screenshot etc.

**Comment by FieryVector** [comment]
Ok, i hope this is all.

Sourcemod Version: 1.8 - build 6040 (Windows)
Metamod Version: 1.10 - build 961 (Windows)
CSGO: Newest version, English
No other mods

## Gamemodes.txt:
(I only play CSGO with this model so i changed gamemodes.txt):
_"de_dust2"
	{
		"nameID"	"#SFUI_Map_de_dust2"
		"name" "de_dust2"
		"imagename" "map-dust2-overall"
		"default_game_type"	"0" // classic
		"default_game_mode" "0" // casual
		"t_arms" "models/player/custom_player/maoling/neptunia/uzume/normal/uzume_arms.mdl"
		"ct_arms" "models/player/custom_player/maoling/neptunia/hatsumi_sega/normal/sega_girl_arms.mdl"
		"t_models"
		{
			"../models/player/custom_player/maoling/neptunia/uzume/normal/uzume" ""
		}
		"ct_models"
		{
			"../models/player/custom_player/maoling/neptunia/hatsumi_sega/normal/sega_girl" ""
		}
	}_

## Console Output:
_Counter-Strike: Global Offensive
Map: de_dust2
Players: 1 (0 bots) / 10 humans
Build: 6974
Server Number: 2

No pure server whitelist. sv_pure = 0
SignalXWriteOpportunity(3)
PutClientInServer: no info_player_start on level
Real Reaper951 connected.
Communications with routing cluster 'cant' established.
0:  Reinitialized 2 predictable entities
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
SignalXWriteOpportunity(3)
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Missing weapon hand bone attachment for player model.
Bad ragdoll for error.mdl
Bad ragdoll for error.mdl
Bad ragdoll for error.mdl
Bad ragdoll for error.mdl
Bad ragdoll for error.mdl_

## Screenshots:
![20171226172214_1](https://user-images.githubusercontent.com/8775780/34360540-f3f41248-ea61-11e7-8bf4-8df0e18b298f.jpg)
![20171226172229_1](https://user-images.githubusercontent.com/8775780/34360544-f622c8ca-ea61-11e7-89bf-f8a1f6504f74.jpg)

**Comment by rogeraabbccdd** [comment]
Models are work for me. I think you should use some plugins to make players download the models.

**Comment by Kxnrl** [comment]
Your kv has syntax error..
Example
``` text
"t_models"
{
"models/player/custom_player/maoling/neptunia/uzume/normal/uzume" ""
}
```

**Comment by rogeraabbccdd** [comment]
Maybe that's why your models are not downloaded or precached.

**Comment by FieryVector** [comment]
Thanks to your help I was able to fix it now for 50%. 
The models are working now, but the arms still not. Since the game always loads the default models its hard to find the error. I still try to find the problem by trying different paths.

And for the models I needed to change the path to:
_"t_models"
			{
				"custom_player/maoling/neptunia/blanc/hdd/whiteheart" ""
			}_
My game use "csgo/models/player/*" as root path, but only for the models, not for the arms.
